src/front/app/__test__/Input.test.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { shallow, render, mount } from 'enzyme';
+import Input from '../components/globals/Input';
+
+describe('(Component) Input', () => {
+  it('Should render', () => {
+    const wrapper = mount(
+      <Input name="random" />
+    );
+    expect(wrapper.find(Input)).toHaveLength(1);
+  });
+
+  it('Should render its children', () => {
+    const wrapper = shallow(
+      <Input name="random">
+        <option value="value">Text</option>
+      </Input>
+    );
+
+    expect(wrapper.contains(<option value="value">Text</option>)).toBe(true);
+  });
+
+  it('Should render with custom classes', () => {
+    const wrapper = render(
+      <Input name="random" className="my-class" />
+    );
+
+    expect(wrapper.attr('class')).toBe('form-control my-class');
+  });
+
+  it('Should add the inputType whenever provided', () => {
+    const wrapper = render(
+      <Input name="random" inputType="number" />
+    );
+
+    expect(wrapper.attr('type')).toBe('number');
+  });
+
+  it('Should render an input by default', () => {
+    const wrapper = mount(
+      <Input name="random" />
+    );
+
+    expect(wrapper.find('input')).toHaveLength(1);
+    expect(wrapper.find('select')).toHaveLength(0);
+    expect(wrapper.find('textarea')).toHaveLength(0);
+  });
+
+  it('Should render a select when provided with children', () => {
```

Comment:
Textarea have children too right? if so, don't default to anything here. If the user specifies a select, then do so.